docs/v3/identity.md

```diff
@@ -0,0 +1,261 @@
+
+When a user messages another user by name, the app will use XMTP to resolve the name to an Inbox ID, starting at the bottom of the diagram and ending at the top.
+
+When a participant is rendered in an app's UX, resolving the name from the Inbox ID will begin from the Inbox ID at the top of the tree and proceed to the name at the bottom.
+
+### Recovery Wallet
+
+A recovery wallet is a single wallet in the Inbox that can manage any object in the Inbox. The first wallet used to create an Inbox is designated as the recovery wallet.  Users can change their recovery wallet to a different wallet, as long as the recovery wallet assents to the change.
+
+By default, a recovery wallet is not addressible for messages.  A recovery wallet can ALSO be an associated wallet, making it addressible for messages, but this is not required.
+
+When an XMTP V2 user upgrades to XMTP V3, their wallet becomes the recovery wallet of the new Inbox, and is associated with the Inbox.
+
+### Associated Wallets
+
+![Inbox Diagram](./img/inbox-diagram.png)
+
+Associated wallets are additional addressible wallets in the Inbox.  An associated wallet can have multiple *installations*.  A user can use the wallet on multiple apps on multiple devices, and each one would have its own installation.  When someone sends a message to the name associated with the wallet, it goes to all the installations in the entire Inbox, not just the installations associated with one wallet.
+
+### Removing Wallets
+
+The recovery wallet has the authority to remove *any* wallet from the Inbox.
+
+When a wallet is removed, all the installations that are registered for that wallet are removed as well.  That means the apps associated with that wallet will no longer be allowed to connect to XMTP (until they add a new wallet and register the new installation).
+
+## Installations
+
+Each wallet needs at least one installation in order to send and receive messages.  An installation is a combination of app and device, and is registered as a child of a single wallet.
+
+### Installation Keys
+
+When you register a new installation, XMTP creates a new Ed25519 key pair that is generated and stored on the device. XTMP uses the Ed25519 signature algorithm and curve25519 elliptic curve. Because XMTP uses Ed25519, the public key is in the format specified in RFC8032.
+
+The public key of the pair is used as the XMTP *signature key*. This is the unique identifier for an XMTP installation.  Messages sent by that installation are encrypted with the private key. Recipient installations use the signature key to decrypt them.
+
+Messages sent to the installation are encrypted using the signature key so only the installation can decrypt them.
```

Comment:
```suggestion
Messages sent to the installation are encrypted using the encryption key so only the installation can decrypt them.
```
Should be encryption key and this line seems to be repeating what's said at the end of last paragraph.